### PR TITLE
refactoring the security check

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: security_check/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def main(argv):
```
Comment (dlorenc) [inline security_check/__init__.py]: Can this file just be empty?

Comment (dlorenc) [inline security_check/__init__.py]: Well, with a copyright header :)

File: security_check/main.py [hunk 2]
```diff
@@ -158,9 +158,19 @@ def _check_vuln_is_valid(vuln):
                  fixed_version)
     return False
 
+def _security_check(image, severity=_MEDIUM, whitelist_file='whitelist.json'):
+    apt_pkg.init()
+    logging.basicConfig(level=logging.DEBUG)
```
Comment (dlorenc) [inline security_check/main.py]: Move the logging config call back up into main.

File: security_check/__init__.py (hunk 1 above)
Comment (sharifelgamal) [inline security_check/__init__.py]: Done.

File: security_check/main.py (hunk 2 above)
Comment (sharifelgamal) [inline security_check/main.py]: Done.

File: security_check/main.py [hunk 3]
```diff
@@ -159,8 +159,19 @@ def _check_vuln_is_valid(vuln):
     return False
 
 
-def _main():
+def _security_check(image, severity=_MEDIUM, whitelist_file='whitelist.json'):
```
Comment (dlorenc) [inline security_check/main.py]: One last nit, you can drop the leading underscore since we're exposing this more broadly now.

**Pull request merged**